File: api/code-samples/python/klay/others/getStakingInfo.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockTag = "latest"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_staking_info(blockTag)

print(json.loads(klay_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/klay/others/getStakingInfo.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const blockNumberOrTag = 'latest'
    
    sdk.klay.getStakingInfo(blockNumberOrTag, {}, (err, data, response) => {
        console.log(data);
    });

}
)()